hera_cal/lstbin.py

```diff
@@ -336,6 +337,12 @@ def lst_bin(data_list, lst_list, flags_list=None, nsamples_list=None, dlst=None,
 
             # sigma clip if desired
             if sig_clip:
+                # flag frequency slices with fewer than min_N data points
+                if flag_below_min_N:
+                    below_min_N_freqs = np.where(f.sum(axis=0) < min_N)[0]
+                    if below_min_N_freqs.size > 0:
+                        d[:, below_min_N_freqs] *= np.nan
```

Comment:
Shouldn't you also be modifying `f` like `f[:, below_min_N_freqs] = True`?

Reply:
Oh yes, I'll add this now